java/libphonenumber/pom.xml

```diff
@@ -42,11 +42,32 @@
       <plugin>
         <artifactId>maven-jar-plugin</artifactId>
         <version>3.0.2</version>
-        <configuration>
-          <archive>
-            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
-          </archive>
-        </configuration>
+        <executions>
+          <execution>
+            <id>default-jar</id>
```

Comment:
Should this have a `phase` and `goals`, and a `classifier` in the `configuration`, to be consistent with the `execution` below?